.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -107,6 +107,22 @@ module frontend_service {
   wait_for_steady_state = local.wait_for_steady_state
 }
 
+resource "aws_secretsmanager_secret" "wmg_config" {
+   name = "corpora/backend/${local.deployment_stage}/wmg_config"
+}
+  
+variable "wmg_secret_string" {
+  default = {
+    tiledb_config_overrides = "{\"py.init_buffer_bytes\": 52428800, \"sm.tile_cache_size\": 134217728, \"sm.mem.total_budget\": 2147483648, \"sm.memory_budget\": 1073741824, \"sm.memory_budget_var\": 2147483648}"
```

Comment:
Couple things to note:
1) We no longer specify `data_path_prefix` in the secret for the `rdev` environments. This should be okay because that is going to be overriden by the `REMOTE_DEV_PREFIX` environment variable which already exists.

2) `rdev` environments were allocated half the tiledb memory compared to `dev,staging,prod`. Here, I just set the memory constraints for all deployment stages, including `rdev`, to that used for `dev,staging,prod` I theoretically could add a conditional statement "if rdev set to half the memory", but I wasn't sure if that was necessary. 

@atolopko-czi any thoughts?